main.py

```diff
@@ -45,9 +45,11 @@ def parse_args():
                         help='enables the use of the grayscale synthetic image for use with non-thresholded images')
     parser.add_argument('--phaseContractImage', type=bool, default=False,
                         help='enables the use of the grayscale synthetic image for phase contract images')
+    parser.add_argument('-a', '--auto_temp', metavar='TEMP', type=int, default=1,
+                          help='auto temperature scheduling for the simulated annealing')
+    parser.add_argument('-t', '--temp', type=float, help='starting temperature for the simulated annealing')
```

Comment:
make this --starttemp, just like endtemp, and perhaps -ts and -te for the short versions.